test/e2e/readme.md

```diff
@@ -6,13 +6,30 @@ End to end tests pull docker images from ghcr.io/celestiaorg/celestia-app. These
 
 ## Usage
 
-E2E tests can be simply run through go tests. They are distinguished from unit tests through an environment variable. To run all e2e tests run:
+**Prerequisite: Requires a kubeconfig file.**
```

Comment:
[question] should we link to the Notion page that describes how to set up a kubeconfig file from this README? We may want to caveat that link with this is only usable by ppl with access to the Celestia scaleway or something like that

Reply:
I was also thinking about that, but I wouldn't link to an internal document in an open-source readme

Reply:
External contributors wouldn't have access to the internal document but it could save future celestia-core contributors some time in figuring out how to set this stuff up correctly

Reply:
Like currently if I'm an internal or external dev, when I read:

> **Prerequisite: Requires a kubeconfig file.**

I have no idea what that means or how to get one. If I instead read:

> Prerequisite: Requires a kubeconfig file. If you're a contributor with access to Celestia Notion / Scaleway, you can set up a kubeconfig file via [these steps](https://www.notion.so/celestiaorg/Configuring-kubeconfig-for-knuui-0848a70da653413c848ac1ba6f5e3957).

then at least we save some time for internal devs.

Reply:
I'm out-numbered so you can ignore my suggestion about including the Notion link.

> However, I'm not sure if there's a Notion page for setting up knuu configs.

There is. I linked it above. Here it is again: https://www.notion.so/celestiaorg/Configuring-kubeconfig-for-knuui-0848a70da653413c848ac1ba6f5e3957